instance/models/mixins/openedx_periodic_builds.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+#
+# OpenCraft -- tools to aid developing and hosting free software projects
+# Copyright (C) 2015-2019 OpenCraft <contact@opencraft.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+Instance app model mixins - Periodic builds
+"""
+
+# Imports #####################################################################
+import datetime
+
+from django.db import models
+
+
+class OpenEdXPeriodicBuildsMixin(models.Model):
+    """
+    Mixin to provide support for periodic builds.
+    """
+
+    class Meta:
+        abstract = True
+
+    build_interval = models.DurationField(
+        default=datetime.timedelta(hours=24),
+        help_text=(
+            "Time interval between periodic builds. Note that this will not be precise; "
```

Comment:
@swalladge It would probably help if you specified the time units to be used here i.e. "Time interval (in hours) between periodic builds...."

Reply:
@pkulkark Ah good point. I will add that to the help text :+1: